css/index.css

```diff
@@ -7,6 +7,7 @@
     --smokewhite-color: whitesmoke;
     --border-color-white: #f3f4f6;
     --secondary-btn-color:#9333EA;
+    --light-gray-color: rgb(216, 216, 216);
```

Comment:
try to maintain consistency either use hex or rgb

Reply:
ok